tests/e2e-upgrade/upgrade-test/01-upgrade-operator.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
-  - command: kubectl apply -f ../../_build/manifests/01-opentelemetry-operator.yaml
+  - script: cd ../../../ && make deploy
   - command: kubectl rollout status -w deployment/opentelemetry-operator-controller-manager -n opentelemetry-operator-system
-  - command: sleep 60s
+  - command: sleep 5s
+  - command: kubectl wait --timeout=5m --for=condition=available deployment opentelemetry-operator-controller-manager -n opentelemetry-operator-system
```

Comment:
IIRC the  `make deploy` already waits for the operator to be ready

Reply:
Yes but, wouldn't it be needed again after the rollout?

Reply:
If I don't remove the rollout, this is needed. Otherwise, we have this error:
```sh
Internal error occurred: failed calling webhook "mopentelemetrycollector.kb.io": failed to call webhook: Post "[https://opentelemetry-operator-webhook-service.opentelemetry-operator-system.svc:443/mutate-opentelemetry-io-v1alpha1-opentelemetrycollector?timeout=10s](https://opentelemetry-operator-webhook-service.opentelemetry-operator-system.svc/mutate-opentelemetry-io-v1alpha1-opentelemetrycollector?timeout=10s)": dial tcp 10.96.63.229:443: connect: connection refused
```
We cannot just deploy the operator and deploy an OTEL Collector instance.

I removed the rollout and added an sleep.